modules/stanford_profile_styles/lib/scss/stanford_profile_styles.scss

```diff
@@ -185,4 +186,21 @@ a {
       }
     }
   }
+
+}
+
+// Menu styles
+nav {
```

Comment:
This selector seems a bit aggressive as it applies to all navs everywhere and I could see a behavior that is different in the future. Could this selector be narrowed to the specific navigation elements you want it for?